feat: make default bash prompt display nonzero exit codes

HastD opened the pull request:
If the previous command's exit code was nonzero, the exit code is displayed inside square brackets in bold and red at the beginning of the bash command prompt. This gives a visual indicator of when a process has silently failed, which is particularly useful for run0 given that it currently will silently fail with exit code 203 for commands where it runs into problems with SELinux.

RoyalOughtness commented on files/system/etc/profile.d/bash-prompt-command.sh:
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
+export PROMPT_COMMAND='__bash_prompt_command 2>/dev/null'
+__bash_prompt_command() {
```
I'm not totally sold on this, it seems like we're hacking around the current run0 selinux issues. 

`echo $?` already exists to check exit codes

HastD replied:
This isn't specific to run0, it's just the example most frequently encountered in secureblue of a command that sometimes fails without clearly indicating failure in the output. I'd argue that the default bash prompt is bad UX and this just fixes that by giving a reliable, clear visual indicator of a command exiting abnormally (which usually indicates some sort of failure).

Yes, you can always do `echo $?`, but users don't always remember to do that (or know about it), and it's especially problematic when a command that you'd expect to have no output on success instead fails silently—even experienced users sometimes make the mistake of assuming the command has succeeded in that situation.

(The specific format of displaying nonzero exit codes in red inside square brackets is already part of the default prompt in fish, by the way.)

HastD commented:
One sec, there's an issue with this conflicting with something `/etc/profile.d/vte.sh` is doing, marking this as draft until I get that sorted out.

HastD committed:
feat: make default bash prompt display nonzero exit codes

If the previous command's exit code was nonzero, the exit code is
displayed inside square brackets in bold and red at the beginning of the
bash command prompt. This gives a visual indicator of when a process has
silently failed.

The file needs to be alphabetically later than vte.sh, since that script
in /etc/profile.d overwrites PROMPT_COMMAND.

Signed-off-by: Daniel Hast <hast.daniel@protonmail.com>

RoyalOughtness committed:
Merge branch 'live' into bash-prompt

RoyalOughtness committed:
Merge branch 'live' into bash-prompt
